cumulusci/tasks/bulkdata/load.py

```diff
@@ -249,6 +256,12 @@ def _query_db(self, mapping):
         """
         model = self.models[mapping.get("table")]
 
+        # Update Account.Name as blank for IsPersonAccount Account records.
+        if mapping[
+            "sf_object"
+        ].lower() == "account" and self._is_person_accounts_enabled(mapping):
+            self._update_person_account_name_as_blank(mapping)
```

Comment:
I agree with David - it could be done cleanly, I think, by adding another generator in the chain within ExtractData's `_import_results()`, and then we don't have to touch the SQL database later. There's some legacy code that still does this, but we'd like to avoid mutating the data store during a load operation.

Reply:
👍   Sounds great and will update.

PS, I merged #1885 into this branch.

Reply:
Setting Account.Name for person account `Account` records is done in the extract task instead of the load task.